autogpt/agent/agent.py

```diff
@@ -195,6 +195,18 @@ def start_interaction_loop(self):
                 elif user_input == "EXIT":
                     print("Exiting...", flush=True)
                     break
+            elif self.next_action_count > 0 and not cfg.continuous_mode:
+                # Print authorized commands left value
+                logger.typewriter_log(
+                    f"{Fore.CYAN}PREAUTHORISED COMMANDS LEFT: {Style.RESET_ALL}{self.next_action_count}"
```

Comment:
"pre-authorized" or "Authorized commands left:" ?

Reply:
I just implemented the suggested changes and filled out the PR properly. Please review. @hdkiller thank you!